### DISCUSSION TITLE
Can I save and reuse request templates in Thunder Client Starter version?

### DISCUSSION THREAD

**Discussion by raymm3852** [thunderclient/thunder-client-support#1640] [Q&A]
I'm using Thunder Client with a Starter version subscription. I've successfully set up a working Claude API request with all the necessary headers and JSON body structure. To save time, I'd like to use this as a template/starting point for future requests without having to re-enter the basic information each time.
I notice that when trying to save to Collections, I see "[Subscription required]". Before I consider upgrading, I want to confirm:

Is saving requests to Collections only available in higher subscription tiers?
If so, are there any workarounds for reusing request configurations in the Starter version?

The key elements I want to reuse are:

POST endpoint URL
Headers configuration
Basic JSON body structure

Would appreciate any guidance on what's possible with the Starter version.

**Comment by rangav** [comment]
Hi,

- Saving requests to collections is available in all paid plans, including the Starter plan.
- We don’t have a concept of templates. However, you can generate a [cURL command](https://docs.thunderclient.com/features/code-snippet) for a request, save the cURL text, and later use the [Import cURL](https://docs.thunderclient.com/features/import#import-curl) feature.
- You can use the collection settings to configure common headers and authentication for all requests within that collection.
